src/transformers/utils/generic.py

```diff
@@ -926,29 +955,134 @@ def can_return_tuple(func):
 
     @wraps(func)
     def wrapper(self, *args, **kwargs):
-        is_requested_to_return_tuple = kwargs.pop("return_dict", True) is False
-        is_configured_to_return_tuple = self.config.use_return_dict is False if hasattr(self, "config") else False
+        return_dict = self.config.return_dict if hasattr(self, "config") else True
```

Comment:
Any reason to use `self.config.return_dict` instead of previously used `self.config.use_return_dict`?
`use_return_dict` takes `torchscript` argument into account as well, so would be breaking. However not sure how common the `torchscript` argument is used.

Reply:
Torchscript is deprecated, and the tests seemed to work a lot better on return_dict rather than use_return_dict!